plugins/ircbridge/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "ircbridge",
-    "version": "0.1.0",
+    "version": "1.0.0",
```

Comment:
Why the mayor version bump in this pr?

Reply:
Did this before I fixed the configuration format compatibility again